src/assets/style/stripView.css

```diff
@@ -0,0 +1,123 @@
+.stripView {
+    height: calc(100% - 36px);
+    width: 400px;
+    position: absolute;
+    top: 0;
+    right: 0;
+    margin: 0 auto;
+    font-family: Monospace, serif;
+    font-size: 10px;
```

Comment:
Can we do 12px here? Slightly oversized to scale, but makes it much easier to read I think.